File: printing_quotes.go
package main

import (
	"bufio"
	"fmt"
	"os"
	"strings"
)

func main() {
	reader := bufio.NewReader(os.Stdin)

	fmt.Print("What is the quote? ")
	quoteInput, _ := reader.ReadString('\n')
	quoteInput = strings.TrimSpace(quoteInput)

	fmt.Print("Who said it? ")
	author, _ := reader.ReadString('\n')
	author = strings.TrimSpace(author)

	response := fmt.Sprintf("%s says, \"%s\"", author, quoteInput)

	fmt.Println(response)
}

File: printing_quotes.exs
defmodule PrintingQuotes do
  def run do
    quote_input = IO.gets("What is the quote? ") |> String.trim()
    author = IO.gets("Who said it? ") |> String.trim()

    response = "#{author} says, \"#{quote_input}\""

    IO.puts(response)
  end
end

PrintingQuotes.run()
